### PR TITLE
Implemented GetMonthlyCost method

### PR DESCRIPTION
**Pull request by kgrontis**
PR targets issue no. #3 

Changes Added:

- Implemented the `GetMonthlyCost` method in the `SubscriptionsCalculator` class.
- Added unit tests for the `GetMonthlyCost` method.

Details:

- The `GetMonthlyCost` method accepts an `IEnumerable<Subscription>` parameter representing the subscriptions for which the monthly cost needs to be calculated.
- The method handles edge cases by checking for null or empty subscriptions. If the subscriptions parameter is null or contains no elements, the method returns 0 as the monthly cost.
- Tests cover scenarios such as empty subscriptions, null subscriptions, a single subscription, and multiple subscriptions with varying costs.

### CODE REVIEW

File: subtrack.MAUI/Services/SubscriptionsCalculator.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using subtrack.DAL.Entities;
+
+namespace subtrack.MAUI.Services;
+
+public static class SubscriptionsCalculator
+{
+    public static decimal GetMonthlyCost(IEnumerable<Subscription> subscriptions)
```
Comment (chrisK00) [inline subtrack.MAUI/Services/SubscriptionsCalculator.cs]: I'd prefer if it would be an extension method instead, so you can just call `subscriptions.GetMonthlyCost()`

**Review by chrisK00 — APPROVED** [review]
Awesome test coverage and invalid input handling

File: subtrack.MAUI/Services/SubscriptionsCalculator.cs (hunk 1 above)
Comment (kgrontis) [reply subtrack.MAUI/Services/SubscriptionsCalculator.cs]: Changed the method to an extension method.
